scripts/docker/alpine/3.10/release.Dockerfile

```diff
@@ -0,0 +1,75 @@
+FROM alpine:3.10
+
+RUN apk update \
+    && apk add --no-cache --upgrade\
+        bison \
+        build-base \
+        cmake \
+        curl \
+        diffutils \
+        file \
+        git \
+        ninja \
+        ronn \
```

Comment:
Is this needed? Does someone wants to compile the man pages in this docker image?

Reply:
Same as above.